docs/src/pages/guides/FHIRPerformanceGuide.md

```diff
@@ -558,7 +558,40 @@ SELECT COUNT(R.VERSION_ID)
 
 In most cases the history queries will execute very quickly. Performance will be slower for cases where a single resource has thousands of versions. To avoid this, ingestion pipelines must ensure they only update a version when necessary.
 
-## 5.4 Search Examples
+## 5.4 Search Performance
```

Comment:
We need to update the "index" at the top of this guide...but it was already stale before this change and so I think we could do it in a separate PR.

Reply:
ok